pkgs/top-level/perl-packages.nix

```diff
@@ -28609,6 +28594,35 @@ with self; {
     };
   };
 
+  Xapian = buildPerlModule rec {
+    pname = "Xapian";
+    version = "1.4.27";
+    src = fetchurl {
+      url = "https://oligarchy.co.uk/xapian/${version}/xapian-bindings-${version}.tar.xz";
```

Comment:
https://oligarchy.co.uk is indeed the current URL on https://xapian.org/download. Apparently the new `Xapian` Perl binding it's not on CPAN (yet).